mmdet/models/detectors/base.py

```diff
@@ -198,6 +198,16 @@ def _parse_losses(self, losses):
         loss = sum(_value for _key, _value in log_vars.items()
                    if 'loss' in _key)
 
+        # If the loss_vars has different length, GPUs will wait infinitely
```

Comment:
suggest to use rank, word_size = get_dist_info(), then use word_size to decide whether to synchronize

Reply:
No need to broadcast the error. At least two GPUs will have their len(log_vars) different from the mean. So users can compare the error logs to determine the missing loss terms.